src/images/shaun-shaker.webp

```diff
@@ -0,0 +1 @@
+
```

Comment:
Please can you replace this image with this one:

![shaun-shaker](https://github.com/shaunsaker/shaun-saker-v2/assets/19655207/8455222c-9a0b-4c87-adb6-794d1751e970)

You'll need to update src/pages/projects/data.tsx as well so that  it imports the new jpg file instead of the webp one.